mdx-gateways/src/test/groovy/com/mx/path/gateway/GatewayConfiguratorTest.groovy

```diff
@@ -19,9 +19,13 @@ import com.mx.testing.*
 import spock.lang.Specification
 
 class GatewayConfiguratorTest extends Specification {
+  def cleanup() {
+    Facilities.reset()
```

Comment:
There was a change in path-core to add default facilities. So I had to add a reset here because it was causing a different unit test to fail (the test checks for 1 eventBus and the result was 2)